bears/haskell/GhcModBear.py

```diff
@@ -0,0 +1,27 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.CabalRequirement import (
+    CabalRequirement)
+
+
+@linter(executable='ghc-mod',
+        output_format='regex',
+        output_regex=r'.+:(?P<line>\d+):(?P<column>\d+):'
+                     r'(?P<message>.+)')
+class GhcModBear:
+    """
+    Syntax checking with ghc for Haskell files.
+
+    See <https://hackage.haskell.org/package/ghc-mod> for more information!
+    """
+
+    LANGUAGES = {'Haskell'}
+    REQUIREMENTS = {CabalRequirement(package='ghc-mod', version='5.6.0')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    ASCIINEMA_URL = 'https://asciinema.org/a/97863'
```

Comment:
Oh wait that is `ghc` output, this is `ghc-mod-5.6.0` output
There is nothing wrong in out Linter - it's their output which is like that :+1: 
```
~/GhcModBearTests/testfile1.hs:2:17:No instance for (Num String) arising from the literal ‘3’In the first argument of ‘putStrLn’, namely ‘3’In the expression: putStrLn 3In an equation for ‘main’: main = putStrLn 3
```